pkg/deployments/tasks/20230316-l2-ve-delegation-proxy/test/test.fork.ts

```diff
@@ -0,0 +1,107 @@
+import hre, { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { Contract } from 'ethers';
+
+import { fp } from '@balancer-labs/v2-helpers/src/numbers';
+import { actionId } from '@balancer-labs/v2-helpers/src/models/misc/actions';
+import * as expectEvent from '@balancer-labs/v2-helpers/src/test/expectEvent';
+
+import { describeForkTest } from '../../../src/forkTests';
+import Task, { TaskMode } from '../../../src/task';
+import { getForkedNetwork } from '../../../src/test';
+import { impersonate } from '../../../src/signers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { ZERO_ADDRESS } from '@balancer-labs/v2-helpers/src/constants';
+import { deploy } from '@balancer-labs/v2-helpers/src/contract';
+
+describeForkTest('L2VotingEscrowDelegationProxy', 'arbitrum', 70407500, function () {
+  let vault: Contract, authorizer: Contract;
+  let veProxy: Contract, nullVotingEscrow: Contract, veDelegation: Contract;
+  let admin: SignerWithAddress, user1: SignerWithAddress, user2: SignerWithAddress;
+
+  let task: Task;
+
+  const GOV_MULTISIG = '0xaf23dc5983230e9eeaf93280e312e57539d098d0';
+
+  const user1VeBal = fp(100);
+  const user2VeBal = fp(200);
+
+  before('run task', async () => {
+    task = new Task('20230316-l2-ve-delegation-proxy', TaskMode.TEST, getForkedNetwork(hre));
```

Comment:
This task name breaks the fork tests tests of `PreseededVotingEscrowDelegation`, which uses the `ve-delegation` alias. Please update that to use the full `20220325-ve-delegation` name there.